### ISSUE TITLE
Unattended updated as a secure default

### ISSUE THREAD

**Issue by jeaye** [zulip/docker-zulip#141]
As a more secure default, each container should be applying any necessary security updates. This is described in detail here: https://help.ubuntu.com/lts/serverguide/automatic-updates.html

In chat, Tim mentioned that this may cause temporary down times, while Posgres updates (for example). Perhaps being more clear that this is happening by sending an administrative email would be helpful.

- [ ] Enable unattended security updates by default
- [ ] Send an admin email before updating

**Comment by timabbott** [comment]
I think my first pass strategy for handling the "error emails" problem would be to modify the content of our error emails to link to a "troubleshooting exceptions" docs page, and have that page document the interaction with unattended-upgrades.  That requires very little engineering work, and would be a nice improvement to the error emails regardless of whether we turn this on.

So I think the issue for docker-zulip can be scoped to just the first two checkboxes: having a (default-on) setting to enable/install unattended-upgrades.

**Comment by galexrt** [comment]
@timabbott @jeaye Guys, you are talking about a container, not a VM.
A container should be rebuilt "every day" or when new updates are available (for the OS base image and application(s)) restarted with the new image. There are tools available to "watch" for changed Docker images (on Docker Hub) and then restart the container (https://github.com/v2tec/watchtower or https://keel.sh/ for Kubernetes).

TL;DR you don't run unattended-upgrades in a container. You update the container image.

**Comment by jeaye** [comment]
Thanks, @galexrt. Still very new to the container world, so I'm in need of reminders like this every now and then.

After reading through the watchtower docs, it looks like something we could pretty trivially add to the `docker-compose.yml` so it's enabled by default on all of the containers. How do we feel about doing that?

**Comment by galexrt** [comment]
@jeaye Yes, that would be a way to do it.
Though there are three points coming from this:

* Creating a scheduled pipeline which will update the image if needed and create a new tag in format `v1.8.1-20180731`.
* Adding what is needed for watchtower to `docker-compose.yml`.
* Linking/Adding docs for using watchtower with docker-zulip.

**Comment by timabbott** [comment]
I'm not convinced this direction is a good path forward for a Docker image; upgrading the image is the preferred way for most folks to update dependencies in a Docker container.
